File: source/server/types/Book.go
package types

// Book 账本信息表
type Book struct {
	Id         int64  `json:"id"`
	BookName   string `json:"bookName"`
	UserId     int64  `json:"userId"`
	CreateDate string `json:"createDate"`
}

type BookQuery struct {
	ID       bool
	BookName bool
	UserId   bool
}

File: webui/src/types/model/book.ts
export interface Book {
  id: string;
  userId: string;
  bookName: string;
  createDate: string;
}

export class BookQuery {
  id?: string;
  userId?: string;
  bookName?: string;
  createDate?: string;
}
/**
* 创建流水的传输实体
*/
export interface CreateBookDto {
  bookName: string;
}
